upgrade/breaking-changes/3-0-0.md

```diff
@@ -0,0 +1,30 @@
+---
+title: 3.0.0
+description: Handle the changes in the 3.0.0 release version of the Telerik UI for .NET MAUI components.
+page_title: Breaking Changes in 3.0.0
+slug: changes-in-3-0-0
+position: 1
+---
+
+# Breaking Changes in 3.0.0
+
+This article lists the breaking changes included in Telerik UI for .NET MAUI 3.0.0 release version.
+
+## Chart
+
+The Chart Series Palette Colors are changed to the latest guidelines.
+
+## CheckBox
+
+* CheckBox is moved to `Telerik.Maui.Controls` namespace.
+* On Android&mdash;The Checkbox UI is updates according to the platform guidelines.
+* On Android&mdash;The touch area arounf the Checkbox is extended.
+* On MacCatalyst&mdash;The size and the corner radius of the Checkbox are updated according to the platform guidelines
```

Comment:
Link?

Reply:
we cannot share the internal design links